packages/modal-ui/src/lib/components/DerivationPath.tsx

```diff
@@ -233,6 +232,29 @@ export const DerivationPath: React.FC<DerivationPathProps> = ({
         </div>
       )}
 
+      {route === "NoAccountsFound" && (
+        <div className="no-accounts-found-wrapper">
+          <p>
+            Can't found any account associated with this Ledger. Please create a
+            new NEAR account on{" "}
+            <a href="https://app.mynearwallet.com/create" target="_blank">
```

Comment:
We'll need to make this dynamic based on the network. Maybe we do need to make `walletUrl` part of the `network` object after all? 😩

Reply:
Did not add it as part of the `network` mostly because we're trying to connect to Ledger but these url(s) are not related to ledger, maybe in the future we could make this url optional as part of the `setupModal()`.

I made this dynamic based on the network it will be either:

https://testnet.mynearwallet.com/create or
https://app.mynearwallet.com/create